actions/md-confluence-sync/action.yml

```diff
@@ -0,0 +1,87 @@
+name: md-confluence-sync
+description: "Sync Markdown documentation to Confluence"
+
+inputs:
+  files:
+    description:
+      "use specified markdown file(s) for converting to html. Supports file
+      globbing patterns"
+    required: true
+  user:
+    description:
+      "Service account email address, should have write access to the given
+      space"
+    required: true
+  token:
+    description: "Service account token for the user above"
+    required: true
+  space:
+    description: "Space to which articles will be synced to"
+    required: true
+  parent:
+    description: "Parent article under which articles will be synced"
+    required: true
+  dry-run:
+    required: false
+    description: "Enable dry-run mode"
+    default: "false"
+  debug:
+    required: false
+    description: "Enabled debug mode"
+    default: "false"
+
+  # grafana inputs (optional)
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+  metrics-id:
+    description:
+      "grafana metrics id, used for continuity of metrics during job name
+      changes  - required if metrics-job-name is passed"
+    required: false
+  gc-host:
+    description: "grafana hostname - required if metrics-job-name is passed"
+    required: false
+  gc-basic-auth:
+    description: "grafana basic auth - required if metrics-job-name is passed"
+    required: false
+  gc-org-id:
+    description:
+      "grafana org/tenant id - required if metrics-job-name is passed"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout repo
+      if: inputs.checkout-repo == 'true'
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: ${{ inputs.checkout-repo-fetch-depth }}
+
+    - name: Setup tools
+      shell: bash
+      run: |
+        go install github.com/kovetskiy/mark@11.1.0
```

Comment:
are we sure that go is present for this to succeed?

Reply:
it's included in the default toolset on github hosted runners: https://github.com/actions/runner-images/blob/main/images/ubuntu/toolsets/toolset-2404.json#L37

Not sure if I need to use setup-go action, but will see once I test it